docs/quickstart.rst

```diff
@@ -0,0 +1,601 @@
+.. eql:section-intro-page:: rocket
+
+==========
+Quickstart
+==========
+
+Welcome to EdgeDB!
+
+This quickstart will walk you through the entire process of creating a simple EdgeDB-powered application: installation, defining your schema, adding some data, and writing your first query. Let's jump in!
+
+
+..    :ref:`Installation <ref_quickstart_install>`
+..    :ref:`Initialize a project <ref_quickstart_createdb>`
+..    :ref:`Set up your schema <ref_quickstart_createdb_sdl>`
+..    :ref:`Insert data <ref_quickstart_insert_data>`
+..    :ref:`Run some queries <ref_quickstart_queries>`
+..    :ref:`Migrate your schema <ref_quickstart_migrations>`
+..    :ref:`Computables <ref_quickstart_computables>`
+..    :ref:`Onwards and upwards <ref_quickstart_onwards>`
+
+.. _ref_quickstart_install:
+
+1. Installation
+===============
+
+First let's install the EdgeDB CLI. Open a terminal and run the appropriate command below.
+
+**macOS/Linux**
+
+.. code-block:: bash
+
+    $ curl https://sh.edgedb.com --proto '=https' -sSf1 | sh
+
+**Windows**
+
+.. code-block::
+
+    # in Powershell
+    PS> iwr https://ps1.edgedb.com -useb | iex
+
+This command downloads and executes a bash script that installs the ``edgedb`` CLI on your machine. You may be asked for your password. Once the installation completes, restart your terminal so the ``edgedb`` command becomes available.
```

Comment:
Perhaps "restart your terminal" should be in bold. Or even add a "note" block here.

Reply:
Done